classification/views/exports/classification_export_formatter_lab_compare.py

```diff
@@ -63,33 +86,109 @@ def extension(self) -> str:
     def header(self) -> List[str]:
         if self.classification_filter.include_sources and len(self.classification_filter.include_sources) == 2:
             lab_names = sorted([str(lab) for lab in self.classification_filter.include_sources])
-            return [delimited_row(['Allele URL', lab_names[0], lab_names[1], 'Difference'], ',')]
+            return [delimited_row(['Allele URL', 'Patient_Id', lab_names[0], lab_names[1], 'Classification',
+                                   'Authorised Date', 'Created Date', 'Other Variables'], ',')]
         else:
             raise ValueError("Must specify 2 labs to compare")
 
     def footer(self) -> List[str]:
         return []
 
     def row(self, allele_data: AlleleData) -> List[str]:
+
+        lab1_comment_data = defaultdict(list)
+        lab2_comment_data = defaultdict(list)
+
         rows: List[str] = []
         lab1 = set()
         lab2 = set()
+        message = ''
+        patient_ids = set()
+        authorised_date_lab1 = set()
+        authorised_date_lab2 = set()
+        created_date_lab1 = set()
+        created_date_lab2 = set()
 
         if not allele_data.allele_id:
             return []
         if self.classification_filter.include_sources:
             lab_name1, lab_name2 = sorted(self.classification_filter.include_sources)
             for cm in allele_data.cms_regardless_of_issues:
-                if cm.lab == lab_name1 and cm.clinical_significance is not None:
-                    lab1.add(cm.get(SpecialEKeys.CLINICAL_SIGNIFICANCE))
-                if cm.lab == lab_name2 and cm.clinical_significance is not None:
-                    lab2.add(cm.get(SpecialEKeys.CLINICAL_SIGNIFICANCE))
+                authorised_by = cm.get('curation_verified_by', '')
+                authorised_date = cm.get('curation_verified_date', '')
+                created_by = cm.get('curated_by', '')
+                created_date = cm.get('curation_date', '')
```

Comment:
Instead of "curation_date" use SpecialEKeys.CURATION_DATE
Instead of "curation_verified_by" use SpecialEKeys.CURATION_VERIFIED_DATE
Instead of "patient_id" use SpecialEKeys.PATIENT_ID